tests/test_util.py

```diff
@@ -281,33 +239,22 @@ def write_cloud_fp(bucket, name, fp):
     return gcs_utils.upload_object(bucket, name, fp)
 
 
-def populate_achilles(hpo_bucket, hpo_id=FAKE_HPO_ID, include_heel=True):
+def populate_achilles(hpo_id=FAKE_HPO_ID, include_heel=True):
     from validation import achilles, achilles_heel
     import app_identity
 
     app_id = app_identity.get_application_id()
```

Comment:
@mark-velez this app_id can be used in place of `project_id`.

Reply:
That's what is happening in the line below..?

Reply:
yes, my bad thats' true. All the tests were passing. Thanks for updating this.